src/Message.vue

```diff
@@ -3,14 +3,14 @@
     <div
       class="sc-message--content"
       :class="{
-        sent: message.author === 'me',
-        received: message.author !== 'me' && message.type !== 'system',
+        sent: message.author === myId,
```

Comment:
hey. This is really good work! I want to use it in my project but could not figure out how and where should I make these changes? I mean I have just <vue-beautiful-component /> . Where should I add myId? Thanks!

Reply:
@oldner I'm not sure I understand you question. This is my comment to my own pull request, so it would be easier to review. This pull request was never merged. In my repository `myId` value gets specified like this:
```
<template>
  <div>
    <beautiful-chat
      ...
      :myId="12345" 
      ...
```